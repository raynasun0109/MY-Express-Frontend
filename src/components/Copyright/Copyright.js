import Typography from '@mui/material/Typography';

export default function Copyright(props) {
    return (
      <Typography className="copyright_container" variant="body2" color="text.secondary" align="center" {...props}>
        {'Copyright Â© '}
       
          MY Shopaholic
        {' '}
        {new Date().getFullYear()}{'.'}
      </Typography>
    );
  }