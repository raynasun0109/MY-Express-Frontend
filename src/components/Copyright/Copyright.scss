@media only screen and (min-width: 1024px) {
}
@media only screen and (max-width: 1023px) {
}
@media only screen and (max-width: 767px) {
}
@media only screen and (max-width: 414px) {
}