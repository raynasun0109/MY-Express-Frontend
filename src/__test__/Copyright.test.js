import renderer from 'react-test-renderer';
import Copyright from '../components/Copyright/Copyright.js';

it('Renders Copyright correctly', () => {
    const tree = renderer
      .create(<Copyright />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });