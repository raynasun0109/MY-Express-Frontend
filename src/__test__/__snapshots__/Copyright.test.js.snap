// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Copyright correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter copyright_container css-1vl95ex-MuiTypography-root"
>
  Copyright Â© 
  MY Shopaholic
   
  2023
  .
</p>
`;
